temporal/t.rast.univar/t.rast.univar.py

```diff
@@ -71,13 +76,15 @@ def main():
     import grass.temporal as tgis
 
     # Get the options
+    options, flags = gs.parser()
```

Comment:
This should be in main, yes, but in the same order as when outside of it, so before the import of grass.temporal.

Reply:
Thanks, that fixed the failing test.